### ISSUE TITLE
Check whether there’s an alternative for people who cannot see maps

### ISSUE THREAD

**Issue by Angelsaur** [alan-turing-institute/AutSPACEs#64]
### Summary

Some websites contain interactive maps.

It’s very hard to make a map itself accessible to people who have visual impairments using some sorts of assistive technology. At the least, you should provide an alternative for users who cannot use the map.

### Deadline 

20/02/2021

### What needs to be done?

It’s very hard to make a map itself accessible to people who have visual impairments using some sorts of assistive technology. At the least, you should provide an alternative for users who cannot use the map.

The Pension Wise website does this well. For instance, when you search for your nearest contact centre, you’re presented not only with a map you could use to navigate, but also with a text address any user could access.

Check any maps on your website to see whether you’re providing alternative routes for users who cannot use the map.

**Comment by gedankenstuecke** [comment]
Made issue #369 to create an accessibility checklist that can be used for the MVP and going forward. Closing this along with other accessibility checklist items for now.
